viper/compile_lll.py

```diff
@@ -173,21 +176,22 @@ def compile_to_assembly(code, withargs=None, break_dest=None, height=0):
         o.extend(['DUP2'])
         # Stack: num num bound
         if code.value == 'uclamplt':
-            o.extend(["LT", 'ISZERO', 'PC', 'JUMPI'])
+            o.extend(["LT", 'ISZERO'])
         elif code.value == "clamplt":
-            o.extend(["SLT", 'ISZERO', 'PC', 'JUMPI'])
+            o.extend(["SLT", 'ISZERO'])
         elif code.value == "uclample":
-            o.extend(["GT", 'PC', 'JUMPI'])
+            o.extend(["GT"])
         elif code.value == "clample":
-            o.extend(["SGT", 'PC', 'JUMPI'])
+            o.extend(["SGT"])
         elif code.value == 'uclampgt':
-            o.extend(["GT", 'ISZERO', 'PC', 'JUMPI'])
+            o.extend(["GT", 'ISZERO'])
         elif code.value == "clampgt":
-            o.extend(["SGT", 'ISZERO', 'PC', 'JUMPI'])
+            o.extend(["SGT", 'ISZERO'])
         elif code.value == "uclampge":
-            o.extend(["LT", 'PC', 'JUMPI'])
+            o.extend(["LT"])
         elif code.value == "clampge":
-            o.extend(["SLT", 'PC', 'JUMPI'])
+            o.extend(["SLT"])
+        o.extend(['PC', 'JUMPI'])
```

Comment:
@DavidKnott by extension shouldn't this also be using REVERT instead?

Reply:
Eventually yes, trying to keep the PRs small 😄

Reply:
Sure, makes sense. Created https://github.com/ethereum/viper/issues/527 so we don't loose track.